code/game/jobs/job_controller.dm

```diff
@@ -511,9 +511,7 @@ var/global/datum/controller/occupations/job_master
 		job.introduce(H, (alt_title ? alt_title : rank))
 	else
 		to_chat(H, "<B>You are the [alt_title ? alt_title : rank].</B>")
-		to_chat(H, "<b>As the [alt_title ? alt_title : rank] you answer directly to [job.supervisors]. Special circumstances may change this.</b>")
-		if(job.req_admin_notify)
-			to_chat(H, "<b>You are playing a job that is important for Game Progression. If you have to disconnect, please notify the admins via adminhelp.</b>")
+		to_chat(H, "<b>As the [alt_title ? alt_title : rank]. Special circumstances may change this.</b>")
```

Comment:
'if(job) else job.req_admin_notify` always runtimes

Reply:
It was probably not supposed to be indented under the else

Reply:
it just seems that this would make the message when you spawn in "as the cargo technician. special circumstances may change this."
just want to make sure spawning stationside with a job doesnt make nervegas

Reply:
You're right. I wasn't even paying attention to this message. Why change it?

Reply:
```
/datum/job/proc/introduce(mob/M, job_title)
	if(!job_title)
		job_title = title
	log_admin("([M.ckey]/[M]) started the game as a [job_title].")
	to_chat(M, "<B>You are the [job_title].</B>")
	to_chat(M, "<b>As the [job_title] you answer directly to [src.supervisors]. Special circumstances may change this.</b>")

	if(src.req_admin_notify)
		to_chat(M, "<b>You are playing a job that is important for Game Progression. If you have to disconnect, please notify the admins via adminhelp.</b>")
```

Everything is written under here